cmd/storage.go

```diff
@@ -50,10 +52,20 @@ var storageDeleteCmd = &cobra.Command{
 		checkError(err, "")
 		projectName := project.GetCurrent(client)
 		componentName := getComponent(client, storageComponent, applicationName, projectName)
-
-		err = storage.Remove(client, storageName, componentName, applicationName)
-		checkError(err, "failed to delete storage")
-		fmt.Printf("Deleted %v from %v\n", storageName, componentName)
+		var confirmDeletion string
+		if storageForceDeleteflag {
+			confirmDeletion = "y"
+		} else {
+			fmt.Printf("Are you sure you want to delete the storage: %v? [y/N] ", storageName)
```

Comment:
It would be good if this would also show if the storage is mounted or not and if yes than to what component.

If not mounted:
`Are you sure you want to delete the storage %v?`

if mounted:
`Are you sure you want to delete the storage %v mounted to %s in  %s component? `
For example:
`Are you sure you want to delete the storage data mounted to /data in backend component? `